content/docs/configuration/venafi.md

```diff
@@ -137,13 +137,13 @@ In order to set up a Venafi Trust Protection Platform `Issuer`, you must first
 create a Kubernetes `Secret` resource containing your Venafi TPP API
 credentials.
 
-NOTE: For TPP >= 19.2 use Access Token Authentication
-and for older versions of TPP, use username / password authentication.
+NOTE: Starting with TPP 22.2, the username and password authentication will
```

Comment:
Should note that in 22.3 API Key authentication is completely removed!

But using user/pass doesn't mean necessarily using API Key no? unless that's how it's implemented in cert-manager

Reply:
I have the same question as @SpectralHiss. I was under the impression we didn't use API keys and cert-manager used the user / pass to hit the Oauth API.

Reply:
Yeah when I read the initial description here, thats the thought that came to my mind. Saying user/pass doesn't mean you go to the api key. You still use a user pass to hit the oAuth token endpoint, but its the actual auth method that has changed. Simply put the endpoint of api key: "vedsdk/authorize" is no longer accessible, instead the endpoint of Oauth: "vedauth/authorize/oauth" is the only available option in 22.3